pyaedt/application/AnalysisTwinBuilder.py

```diff
@@ -89,9 +87,33 @@ def existing_analysis_sweeps(self):
 
     @property
     def modeler(self):
-        """Design oModeler."""
+        """Design Modeler.
+
+        Returns
+        -------
+        :class:`pyaedt.modeler.schematic.ModelerTwinBuilder`
+        """
+        if self._modeler is None:
+            from pyaedt.modeler.schematic import ModelerTwinBuilder
+
+            self._modeler = ModelerTwinBuilder(self)
         return self._modeler
 
+    @property
+    def post(self):
+        """Design Post Processor.
+
+                Returns
+                -------
+                :class:`pyaedt.modules.PostProcessor.CircuitPostProcessor
+        `
```

Comment:
```suggestion
    def post(self):
        """Design Post Processor.

            Returns
            -------
            :class:`pyaedt.modules.PostProcessor.CircuitPostProcessor`
```

Reply:
I think Post Processor should be one word as it was earlier.